src/pkg/services/m365/api/users.go

```diff
@@ -268,6 +269,18 @@ func EvaluateMailboxError(err error) error {
 	return err
 }
 
+// IsErrMailboxNotFound inspects the secondary errors inside MailboxInfo and
+// determines whether the resource has a mailbox.
+func IsErrMailboxNotFound(errs []error) bool {
```

Comment:
Can you please move this to the graph package, so that it can be co-located with the other errors comparators?

Reply:
This is not a graph error. It's defined in api.users